godot-codegen/src/util.rs

```diff
@@ -51,6 +51,12 @@ pub fn make_enum_definition(enum_: &Enum) -> TokenStream {
                     Self { ord: self.ord | rhs.ord }
                 }
             }
+
+            impl Default for #enum_name {
+                fn default() -> Self {
+                    Self { ord: 0 }
+                }
+            }
```

Comment:
I'd like to not provide enum defaults yet, especially since `ord: 0` is not necessarily a valid one.

For testing purposes, could you select one of the existing enumerators?

Reply:
It's only for bitfields, otherwise there's no sensible way to create an empty bitfield. I'm adding a comment.

Reply:
Ah, I see!

Wouldn't it be easier to uncomment this? That would be the purpose, I just didn't activate it until we need it:
https://github.com/godot-rust/gdextension/blob/eb5d43b3e021e1eb1b67661d91318660cd54794c/godot-codegen/src/util.rs#L44-L46

Reply:
Reverted, was only used in an itest that's now gone.